### DISCUSSION TITLE
Online discussion chat/voice/video and help for anyone interested

### DISCUSSION THREAD

**Discussion by MykolaGolubyev** [testingisdocumenting/webtau#1163] [Announcements]
Hello,
Let me know if you need any help achieving your testing goals!
Open for async communication here or emails, chats, voice or video calls. 

I am curios to learn about your use cases, which may give me new ideas to implement!

**Comment by MykolaGolubyev** [comment]
@aplatypus let me know what do you think

**Reply by aplatypus** [reply]
* Yes I can see the value here.  
* I think using Stackoverflow Q&A vehicle works well so far, because it provides a record of 'matters arising' as we go.
* At this point my testing cycle is done for the current module -- Which is an example of how this kind of seeking goes.  Off-and-on for each new use case 
  * The PR is in review, so there may be follow-up
*  The recent use for WebTau was by way of a 'pilot' or feasibility exercise to gauge how well WebTau fits with the CI and JUnit support we currently have.
* And it did well, I feel it is quite suitable.  
* It might be a good thing to set-up a room on Matrix where people can chat, I still think the Q&A is a better structure and helpfully self-documenting.

**Reply by MykolaGolubyev** [reply]
@aplatypus was wondering if you have accumulated new interesting use cases or suggestions

**Comment by MykolaGolubyev** [comment]
Discord server is created: https://discord.gg/Z5yjatDy

**Reply by MykolaGolubyev** [reply]
@aplatypus FYI
